### PR TITLE
Fix regression introduced by commit 8ae91b5.

### PR DESCRIPTION
**Pull request by bsmiles32**

### CODE REVIEW

File: src/r4300/new_dynarec/new_dynarec.c [hunk 1]
```diff
@@ -1149,9 +1149,12 @@ static void ll_kill_pointers(struct ll_entry *head,int addr,int shift)
        (((ptr-(u_int)base_addr-MAX_OUTPUT_BLOCK_SIZE)>>shift)==((addr-(u_int)base_addr)>>shift)))
     {
       inv_debug("EXP: Kill pointer at %x (%x)\n",(int)head->addr,head->vaddr);
+      u_int host_addr=(int)kill_pointer(head->addr);
       #if NEW_DYNAREC == NEW_DYNAREC_ARM
-        u_int host_addr=(int)kill_pointer(head->addr);
```
Comment (Narann) [inline src/r4300/new_dynarec/new_dynarec.c]: Why this wasn't working?

**Comment by bsmiles32** [comment]
As explained by @Gillou68310 , kill_pointer call has to be done for all platforms not just ARM. The returned value however is only needed  by the ARM. Since it wasn't used by the x86 it generated warnings on x86 but not on ARM.

**Comment by Narann** [comment]
I just realize you did a [PR and merge instantly](https://github.com/mupen64plus/mupen64plus-core/pull/105). I suggest to let one tiny day for PRs related to dynarec code to maybe let some others test it as we have no way yet to test multiple build options.

**Comment by bsmiles32** [comment]
see the name of the PR :)

**Comment by Narann** [comment]
What I mean is that if we had wait a tiny day before merge the previous PR we maybe wouldn't have to create another PR to fix it. :D

**Pull request merged**